l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1633,14 +1725,20 @@ def _set_invoice_lines(
     # move_id
     # account_id
     def set_invoice_line_ids(
-        self, FatturaBody, credit_account_id, partner, wt_founds, invoice
+        self,
+        FatturaBody,
+        account_id,
+        partner,
+        wt_founds,
+        invoice,
+        invoice_type="purchase",
```

Comment:
Aggiungendo questo parametro rompe l10n_it_fatturapa_in_rc in quanto nel modulo è esteso il wizard e questo metodo non avendo il parametro aggiuntivo riporta "expected 6 arguments got 7"

Reply:
aggiunto parametro mancante